lambda/resolveSingleCellGeoLocation.ts

```diff
@@ -64,20 +62,17 @@ const serviceToken = once(
 					Authorization: `Bearer ${apiKey}`,
 				},
 			},
+			Type.Object({
+				token: Type.String(),
+			}),
 		)
-		if (!res.ok) {
-			const body = await res.text()
-			console.error('request failed', {
-				body,
-				status: res.status,
-			})
-			throw new Error(`Acquiring service token failed: ${body} (${res.status})`)
-		}
-		const { token } = (await res.json()) as {
-			createdAt: string // e.g. '2022-12-19T19:39:02.655Z'
-			token: string // JWT
+
+		if ('error' in maybeResult) {
+			log.error('request failed', { error: maybeResult.error })
+			throw new Error(`Acquiring service token failed: ${maybeResult.error}`)
```

Comment:
**comment:** this swallows the validation errors (because the error is stringified). But OK in this case because error is logged and function is only used in lambda.